TCAT/Views/PhraseLabelFooterView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  PhraseLabelFooterView.swift
+//  TCAT
+//
+//  Created by Matt Barker on 4/29/18.
+//  Copyright © 2018 cuappdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class PhraseLabelFooterView: UITableViewHeaderFooterView {
+
+    var label: UILabel = {
+        let label = UILabel()
+        label.text = ""
+        label.font = UIFont(name: Constants.Fonts.SanFrancisco.Regular, size: 12)
+        label.textColor = .lightGray
+        label.numberOfLines = 0
+        label.textAlignment = .left
+        label.lineBreakMode = .byWordWrapping
+        return label
+    }()
```

Comment:
better to move this styling and initialization of label into `init(reuseIdentifer:)`. It better tells the story of initialization